code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -46,3 +50,6 @@ Bonus
 				M.emote("sniff")
 		else
 			M.emote("sneeze")
+			if(infective && !(A.spread_flags & DISEASE_SPREAD_FALTERED) && prob(40))
+				addtimer(CALLBACK(A, /datum/disease/.proc/spread, 4), 20)
+				M.visible_message("<span class='danger'>[M] sneezes, letting loose a spray of mucous!</span>")
```

Comment:
Again, the *sneeze is enough, rmeove this message.

Reply:
I felt it was important to make it distinct when it would spread, but Ok